core/src/test/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -5818,6 +5818,21 @@ public void subTestIntervalDayFailsValidation() {
     assertTrue(sqlNodeVisited.getKind() == SqlKind.INSERT);
   }
 
+  @Test void testSqlInsertSqlBasicCallToString() throws Exception {
+    final String sql = "insert into emps select * from emps";
+    final SqlNode sqlNode = getSqlParser(sql).parseStmt();
+    final SqlNode sqlNodeVisited = sqlNode.accept(new SqlShuttle() {
+      @Override public SqlNode visit(SqlIdentifier identifier) {
+        return new SqlIdentifier(identifier.names,
+            identifier.getParserPosition());
+      }
+    });
+    final String str = "INSERT INTO `EMPS`\n"
```

Comment:
what about one more test regarding the actual execution?

Reply:
Thanks, update the code.